buildbot/master/files/config/master.cfg

```diff
@@ -231,6 +232,16 @@ arm64_factory = create_servo_factory([
     steps.Compile(command=["./mach", "build", "--rel", "--target=aarch64-unknown-linux-gnu"], env=arm64_compile_env),
 ])
 
+windows_compile_env = dict({'PATH': r'C:\msys64\mingw64\bin;C:\msys64\usr\bin\C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Amazon\cfn-bootstrap',
+                            'MSYSTEM': 'MINGW64',
+                            'MSYS': 'winsymlinks=lnk'},
+                           **common_test_env)
```

Comment:
Luckily, I have 100GB of disk on this machine :-) AppVeyor's cache is only 500MB.

That said, we have *no* cache reduction on any of the machines, so they will all slowly accrete more cargo packages & rust/cargo binaries over time until they fill all available disk. Opening #321